features/security/src/main/feature/feature.xml

```diff
@@ -560,6 +560,7 @@
         <feature>security-handler-api</feature>
         <feature>javax-validation</feature>
         <bundle>mvn:ddf.security.idp/security-idp-plugin-subjectconfirmation/${project.version}</bundle>
+        <bundle>mvn:ddf.security.idp/security-idp-plugin-requestsubjectvalidator/${project.version}</bundle>
```

Comment:
❓ I wonder if we shouldn't refactor all the `security-idp-plugin` bundles (and api bundle) into a new feature to keep the `security-idp` feature a little more concise.